packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantAvailability/RestaurantAvailability.vue

```diff
@@ -84,17 +84,15 @@ export default {
     margin-right: spacing(b);
 }
 
+.c-restaurantCard-availability > p + span:before {
```

Comment:
is there any way to leverage class selectors here? is this for `c-restaurantCard-availability-message` or a different span?

Reply:
This is the only way I could get this to consistently work

Reply:
I am leaning on @jamieomaguire's side here, is it possible to use class name value for the element, rather than increasing style specificity?

Reply:
Sorry both, yes updated now, had a little 🧠  block this morning 😬 😀